QuestionsforKaida_s2.md

```diff
@@ -0,0 +1,6 @@
+- How can I refactor the code without repeating the  footer and nav for each page?
```

Comment:
This isn't something you can do with pure HTML.  It's the kind of thing you could do with templating or components (frameworks like Flask have support for templates, React has support for components) but have to just copy/paste with pure HTML.